src/prettytable/prettytable.py

```diff
@@ -206,7 +206,7 @@ def __init__(self, field_names=None, **kwargs) -> None:
         self._start = kwargs["start"] or 0
         self._end = kwargs["end"] or None
         self._fields = kwargs["fields"] or None
-        self._none_format = {}
+        self._none_format: dict[None, None] = {}
```

Comment:
This can at least also have a string for keys and values:

```suggestion
        self._none_format: dict[str | None, str | None] = {}
```